packages/twenty-front/src/modules/favorites/components/FavoriteFoldersMultiSelect.tsx

```diff
@@ -170,35 +168,29 @@ export const FavoriteFoldersMultiSelect = ({
         <DropdownMenuItemsContainer>
           <StyledItemsContainer>
             {showNoFolderOption && (
-              <MenuItem
-                key={NO_FOLDER_ID}
-                onClick={() => toggleFolderSelection(NO_FOLDER_ID)}
-                LeftIcon={() => (
-                  <StyledCheckbox
-                    checked={favoriteFoldersMultiSelectChecked.includes(
-                      NO_FOLDER_ID,
-                    )}
-                  />
+              <MenuItemMultiSelect
+                key={`menu-${NO_FOLDER_ID}`}
+                onSelectChange={() => toggleFolderSelection(NO_FOLDER_ID)}
+                selected={favoriteFoldersMultiSelectChecked.includes(
+                  NO_FOLDER_ID,
                 )}
                 text="No folder"
+                className=""
               />
             )}
             {showNoFolderOption && filteredFolders.length > 0 && (
               <StyledDropdownMenuSeparator />
             )}
             {filteredFolders.length > 0
               ? filteredFolders.map((folder) => (
-                  <MenuItem
-                    key={folder.id}
-                    onClick={() => toggleFolderSelection(folder.id)}
-                    LeftIcon={() => (
-                      <StyledCheckbox
-                        checked={favoriteFoldersMultiSelectChecked.includes(
-                          folder.id,
-                        )}
-                      />
+                  <MenuItemMultiSelect
+                    key={`menu-${folder.id}`}
+                    onSelectChange={() => toggleFolderSelection(folder.id)}
+                    selected={favoriteFoldersMultiSelectChecked.includes(
+                      folder.id,
                     )}
                     text={folder.name}
+                    className=""
```

Comment:
can you explain why you need an empty className?

Reply:
Its the way MenuItemMultiSelect is written and used elsewhere, should I make classname optional to be passed?
That way this wont be a problem?

Reply:
you don't need to provide the empty string className here isn't it?

Reply:
Ah, good point